airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/job_error_reporter/SentryExceptionHelper.java

```diff
@@ -23,10 +23,9 @@
 @SuppressWarnings("PMD.AvoidLiteralsInIfCondition")
 public class SentryExceptionHelper {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(SentryExceptionHelper.class);
+  public record SentryParsedException(String platform, List<SentryException> exceptions) {}
```

Comment:
Maybe an enum of the platforms is more appropriate here?

Reply:
👍🏻 done in [85cbf67](https://github.com/airbytehq/airbyte/pull/16906/commits/85cbf67a077a0835cccfc3e72622a9d9c32a86b7)